code401Challenges/src/main/java/code401Challenges/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,77 @@
+package code401Challenges.linkedlist;
+
+import java.util.ArrayList;
+
+public class LinkedList {
+    Node head;
```

Comment:
You should get in the habit of specifying whether an instance variable is public, protected or private